Projects/hCoreUI/Sources/Views/ContractRow.swift

```diff
@@ -110,18 +110,15 @@ private struct ContractRowButtonStyle: SwiftUI.ButtonStyle {
                     )
             }
         } else {
-            hColorScheme(
-                light: hTextColor.Opaque.secondary,
-                dark: hGrayscaleOpaqueColor.greyScale900
-            )
+            hFillColor.Opaque.tertiary
         }
     }
 
     @ViewBuilder var logo: some View {
-        Image(uiImage: hCoreUIAssets.helipadOutlined.image.withRenderingMode(.alwaysTemplate))
+        Image(uiImage: hCoreUIAssets.helipadBig.image.withRenderingMode(.alwaysTemplate))
```

Comment:
replace this line with
hCoreUIAssets.helipadBig.view
You don't need withRenderingMode(.alwaysTemplate) when using template image